packages/homepage/content/docs/2-importing.md

```diff
@@ -32,6 +32,7 @@ We infer sandbox settings based on several files in a repository.
 | Template - React            | If `package.json` dependencies contains `react-scripts`.    |
 | Template - React-Typescript | If `package.json` dependencies contains `react-scripts-ts`. |
 | Template - Svelte           | If `package.json` dependencies contains `svelte`.           |
+| Template - Node             | If `package.json` dependencies contains `ember-cli`.        |
```

Comment:
This really corresponds to the change that will be introduced by https://github.com/codesandbox-app/codesandbox-importers/pull/16.

While this documentation is accurate, it may not be clear to readers why things are set up this way. Ember-CLI apps are just node apps.

**Maintainers:** I could certainly understand wanting to remove this line, in the interest of avoiding potential confusion as to whether people should install ember-cli as a dependency _just_ to get their app recognized as a node sse project. Please let me know how you'd like this handled

Reply:
Yep, we can remove this line as Ember will be a real template now 😄. Could you remove this in combination with the formatting of the new sandboxes modal @SaraVieira?

Reply:
Yes sir !